persistence/mongodb/index.md

```diff
@@ -63,14 +63,22 @@ Note that this disables the ability to use pessimistic locking for sagas which m
 
 ### Shared transactions
 
-NServiceBus supports sharing MongoDB transactions between Saga persistence and business data. The shared transaction can be used to persist document updates for both concerns atomically.
+NServiceBus supports sharing MongoDB sessions between Saga persistence, Outbox storage, and business data. The shared session can be used to persist multiple document updates atomically.
 
 To use the shared transaction in a message handler:
 
 snippet: MongoDBHandlerSharedTransaction
 
 WARN: In order to participate in the shared transaction the MongoDB session must be passed into collection API calls as demonstrated above.
 
+The shared session can also be accessed via dependency injection using the `IMongoSynchronizedStorageSession` interface:
```

Comment:
If I recall correctly we are usually not doing versioned partials for minors right?

Reply:
not sure there is a clear guidance, I think I've seen both. One option would be to just state the minim version (e.g. "Starting from version ..."), use partials or just leave as is since it will become the only supported version anyway.

Reply:
I have several times applied the easy route and just mentioned latest for minor changes. Otherwise the documentation pages become super complex to maintain and at the end of the day rolling forward to a new minor should be fairly straight forward for customers